projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -973,6 +982,18 @@
 				}
 			}
 
+			&.is-non-animated-label .animated-label__label {
```

Comment:
what are these styles for? they don't look specific to the new field.

Reply:
I made them more generic so that in case we fun into a field like this again we can just apply the same styles.

Reply:
The rules are already scoped to `.jetpack-contact-form { ` so we are not adding any new style. It could happen that some other plugin/theme might be adding something that could conflict here. but so could other generic classes. That we have in our code base.

Reply:
Perhaps the classname could be `is-static-label`.

Single/Multiple Choice also have no label animation, so there might also be some usage there. They probably also have some similar CSS somewhere. They do use the `notch` though after the refactor.